plone/namedfile/scaling.py

```diff
@@ -17,7 +17,12 @@
 from zope.traversing.interfaces import ITraversable, TraversalError
 from zope.publisher.interfaces import IPublishTraverse, NotFound
 from zope.app.file.file import FileChunk
-from plone.protect.interfaces import IDisableCSRFProtection
+
+try:
+    # Soft dependency to make this package work without plone.protect
+    from plone.protect.interfaces import IDisableCSRFProtection
+except ImportError:
```

Comment:
IIRC, there was a bigger motion to use ImportError, but we did not discuss this to the end.
Personally, I dislike the pkg_resource way, because I always have to look it up vs using the easy to remember Python syntax for that.
Can you remember, where this discussion took place on github? I remember @gforcada, @davisagli and me were discussing that.